### PR TITLE
Add CRUD helper methods for all mappers

### LINKED ISSUE #7: Write CRUD Operations for all Mappers
Mentors : @aakashrstg00
**Brief Idea**
In Uzay, we have implemented some of the CRUD operations in some of the Mappers as of now. We wish to have CRUD operations implemented in the same existing way using the same existing pattern for the remaining places.

**Required End Results**
All CRUD operations for all the Mappers should be implemented using Mongoose operations. 

**Notes**
The task is to implement these operations which seem useful. It is not 100% necessary that all the CRUD would apply, so you are required to be thoughtful about the design while implementing it. Feel free to discuss doubts here or on Slack Channel/DM.

Please also follow the guidelines mentioned in CONTRIBUTING.md for development.

### PR DESCRIPTION
**Pull request by ashwani99**
Create, read, update, delete options are added for blog post and comment mappers. Create, read and update methods are added for tags and users

Fixes #7

### CODE REVIEW

**Comment by ashwani99** [comment]
@sakshamsaxena Please review

File: mappers/Blog.js [hunk 1]
```diff
@@ -18,4 +24,18 @@ BlogPostMapper.GetBlogByID = function(id) {
 
 };
 
+
+BlogPostMapper.createBlogPost = function(properties) {
+	let post = new BlogPost(properties);
+	post.save();
```
Comment (sakshamsaxena) [inline mappers/Blog.js]: I want the functions to _return_ some sort of promise. It'll be best to use a return statement here with `save()` in such a way that it returns a promise to be used further in Logic etc. Applies to all the `save()` you've used in other mappers.

**Comment by ashwani99** [comment]
@sakshamsaxena I have made the suggested changes

File: mappers/User.js [hunk 2]
```diff
@@ -24,4 +24,13 @@ UserMapper.GetUserById = function(id) {
 
 };
 
+UserMapper.createUser = function(properties) {
+	let user = new User(properties);
+	return tag.save();
```
Comment (IamRaviTejaG) [inline mappers/User.js]: @sakshamsaxena @ashwani99 Please check this!

Comment (ashwani99) [reply mappers/User.js]: @IamRaviTejaG Thanks for pointing that.
@sakshamsaxena I have opened a PR with the fix #20

Comment (sakshamsaxena) [reply mappers/User.js]: Humph. I shouldn't be merging PRs in sleep I guess :smile: Thanks for pointing out!

**Comment by sakshamsaxena** [comment]
Don't worry about conflicts, leave that to me :wink:

**Pull request merged**